native/src/components/CityEntry.tsx

```diff
@@ -40,14 +40,14 @@ const Aliases = styled.View`
   margin: 0 5px;
 `
 
-type PropType = {
+type CityEntryPropsType = {
   city: CityModel
   filterText: string
   navigateToDashboard: (city: CityModel) => void
   theme: ThemeType
 }
 
-class CityEntry extends React.PureComponent<PropType> {
+class CityEntry extends React.PureComponent<CityEntryPropsType> {
```

Comment:
```suggestion
class CityEntry extends React.PureComponent<CityEntryProps> {
```

Reply:
Hopefully also the last remains of the PropsType are renamed. I resolve all these conversations now